cli/src/commands.rs

```diff
@@ -76,17 +71,23 @@ impl Cli {
             Commands::NewPayout {
                 factory_addr,
                 payout_csv,
+                db_deploy,
             } => {
                 let addr = Address::from_str(factory_addr)?;
-                let mut reader = csv::Reader::from_path(payout_csv)?;
-                let mut shares: Vec<U256> = Vec::new();
-                let mut payees: Vec<Address> = Vec::new();
-                for record in reader.deserialize() {
-                    let record: Payment = record?;
-                    let payee = record.payee.parse::<Address>()?;
-                    let share: U256 = (record.shares * ATTO_FIL).into();
-                    payees.push(payee);
-                    shares.push(share);
+
+                println!(" Db Deploy {:#?}", db_deploy);
+                let payees;
+                let shares;
+                if *db_deploy {
+                    (payees, shares) = parse_payouts_from_db().await.unwrap();
+                } else {
+                    (payees, shares) = match payout_csv {
+                        Some(csv_path) => parse_payouts_from_csv(csv_path).unwrap(),
+                        None => {
```

Comment:
@alexander-camuto wondering if my handling here is right. I tried to make this functionality a CLI error instead of a code error itself using `ArgGroup` from `clap` but could not get it working correctly.